### ISSUE TITLE
Unable to create event in additional or shared calendars through SOGo web UI

### ISSUE THREAD

**Issue by bmastenbrook** [mailcow/mailcow-dockerized#4788]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/mailcow/mailcow-dockerized/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... I understand that not following the below instructions will result in immediate closure and/or deletion of my issue.
- [X] ... I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [X] ... I have understood that answers are voluntary and community-driven, and not commercial support.
- [X] ... I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

### Description

There is a bug in SOGo 5.7.1 that prevents users from creating events in any but the first calendar through the SOGo web UI. It's still possible to create events through CalDAV, but it is not working from the browser. The SOGo bug is here: https://bugs.sogo.nu/view.php?id=5581

### Logs

```bash
N/A - this is a known SOGo bug that is affecting my installation.
```

### Steps to reproduce

1. Create a mail user in a domain
2. In the SOGo web UI, add a new calendar in addition to the default personal calendar.
3. Attempt to create a new event in the SOGo web UI. You will only be offered the choice of the personal calendar for the event.

### System information

| Question | Answer |
| --- | --- |
| My operating system | Debian |
| Is Apparmor, SELinux or similar active? | AppArmor |
| Virtualization technology (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** |  KVM |
| Server/VM specifications (Memory, CPU Cores) | 2c, 8GB RAM |
| Docker version (`docker version`) | 20.10.18 |
| docker-compose version (`docker-compose version`) | v2.10.2 |
| mailcow version (```git describe --tags `git rev-list --tags --max-count=1` ```) | 2022-09 |
| Reverse proxy (custom solution) | None |

Output of `git diff origin/master`, any other changes to the code? If so, **please post them**:

`SOGoDomainsVisibility` has been changed in `data/conf/sogo/sogo.conf`.
`ipv6nat-mailcow` has been removed in `docker-compose.yml` (I believe this was done by the setup script).

All third-party firewalls and custom iptables rules are unsupported. **Please check the Docker docs about how to use Docker with your own ruleset**. Nevertheless, iptabels output can help us to help you:

N/A, not a firewall issue.

DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output:

N/A, not a DNS issue.

**Comment by MAGICCC** [comment]
Should be fixed in SOGo 5.7.2, which is not released yet, but we can wait for the release and then close this issue

**Comment by bmastenbrook** [comment]
Sounds good. My intent was to make sure there was a mailcow issue pointing at the SOGo issue so that any other users facing this issue can find this and see that it's a known issue. In the meantime, pulling the set of JS files from master as indicated in the SOGo ticket into the container addressed the issue for my users; just note that in the `sogo-mailcow` container, SOGo is in `/usr/lib` not `/usr/local/lib`.
